src/main/kotlin/uk/gov/gdx/datashare/service/DeathNotificationService.kt

```diff
@@ -145,3 +145,9 @@ data class DeathNotificationDetails(
   @Schema(description = "Whether the deceased was retired", required = false, example = "false")
   val retired: Boolean? = null,
 )
+
+enum class GenderType {
+  Male,
```

Comment:
There's a part of me that feels like enums should all be caps, but I assume this is a serialisation point thing?

Reply:
@olevett updated